canonical-websites/www.canonical.com
# implemented latest footer from vanilla theme

Done:
Implemented the new footer style from the canonical-vanilla-theme

QA:
run the site with all the up-to-date dependencies and ensure the footer looks correct.
